easybuild/easyconfigs/e/Elk/Elk-7.0.12-foss-2020b.eb

```diff
@@ -0,0 +1,60 @@
+easyblock = 'MakeCp'
+
+name = 'Elk'
+version = '7.0.12'
+
+homepage = 'http://elk.sourceforge.net/'
+description = """An all-electron full-potential linearised
+augmented-plane wave (FP-LAPW) code with many advanced features. Written
+originally at Karl-Franzens-Universität Graz as a milestone of the
+EXCITING EU Research and Training Network, the code is designed to be as
+simple as possible so that new developments in the field of density
+functional theory (DFT) can be added quickly and reliably.
+"""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+toolchainopts = {'usempi': True, 'openmp': True, 'opt': True}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = [SOURCELOWER_TGZ]
+checksums = ['9995387c681d0e5a9bd52cb274530b23c0370468b6be86f6c90a6ec445cb8a01']
+
+dependencies = [
+    ('libxc', '5.1.2'),
+    ('Wannier90', '3.1.0'),
+]
+
+
+# make.inc file is generated interactively by "setup" command, creating it here
+prebuildopts = 'echo "F90_OPTS = -std=legacy $FFLAGS" > make.inc && '
+prebuildopts += 'echo "F77_OPTS = -std=legacy $FFLAGS" >> make.inc && '
+prebuildopts += 'echo "LIB_LPK = $LIBLAPACK" >> make.inc && '
+prebuildopts += 'echo "LIB_FFT = $LIBFFT" >> make.inc && '
+prebuildopts += 'echo "LIB_W90 = -lwannier" >> make.inc && '
+prebuildopts += 'echo "LIB_libxc = -lxcf90 -lxc" >> make.inc && '
+prebuildopts += 'echo "SRC_libxc = libxcf90.f90 libxcifc.f90" >> make.inc && '
+prebuildopts += 'echo "SRC_FFT = zfftifc_fftw.f90" >> make.inc && '
+prebuildopts += 'echo "SRC_OBLAS = oblas_stub.f90" >> make.inc && '
+prebuildopts += 'echo "SRC_BLIS = blis_stub.f90" >> make.inc && '
+prebuildopts += 'echo "SRC_MKL = mkl_stub.f90" >> make.inc && '
+
+buildopts = 'all'
+
+runtest = 'test'
+
+parallel = 1
+
+files_to_copy = [(
+    [
+        'src/elk', 'src/spacegroup/spacegroup', 'src/eos/eos',
+        'utilities/elk-bands/elk-bands', 'utilities/elk-optics/elk-optics.py',
+        'utilities/wien2k-elk/se.pl'
+    ], 'bin'
+)]
+
+sanity_check_paths = {
+    'files': ['bin/elk', 'bin/spacegroup', 'bin/eos'],
```

Comment:
@zarybnicky Any sanity check command we can add for `elk`?

Reply:
fixed in 5aab141c21

Reply:
Nice, thanks, I was just in the process of waiting for libxc to build :)